docs/self-managed/identity/deployment/configuration-variables.md

```diff
@@ -134,14 +135,93 @@ steps:
+| `IDENTITY_TENANTS_0_NAME`                    | A human-readable name for the tenant.                                                                                                                                                                                                              | Default       |
+| `IDENTITY_TENANTS_0_TENANTID`                | The tenant ID. The `<default>` tenant is automatically created during Identity startup.                                                                                                                                                            | `<default>`   |
+| `IDENTITY_TENANTS_0_MEMBERS_0_TYPE`          | The type of member to add to the tenant, one of `GROUP`, `USER`, or `APPLICATION`. Additional members can be added by incrementing the second `0` value (for example, `IDENTITY_TENANTS_0_MEMBERS_1_TYPE`.)                                        | `APPLICATION` |
+| `IDENTITY_TENANTS_0_MEMBERS_0_APPLICATIONID` | If adding an **application** as a member of the tenant, the application ID. If additional members have been added to the tenant, ensure the second `0` value matches that of its type (for example, `IDENTITY_TENANTS_0_MEMBERS_1_APPLICATIONID`). | `zeebe`       |
+| `IDENTITY_TENANTS_0_MEMBERS_0_USERNAME`      | If adding a **user** as a member of the tenant, the username. If additional members have been added to the tenant, ensure the second `0` value matches that of its type (for example, `IDENTITY_TENANTS_0_MEMBERS_1_USERNAME`).                    | `user-name`   |
+| `IDENTITY_TENANTS_0_MEMBERS_0_GROUPNAME`     | If adding a **group** as a member of the tenant, the name of the group. If additional members have been added to the tenant, ensure the second `0` value matches that of its type (for example, `IDENTITY_TENANTS_0_MEMBERS_1_GROUPNAME`).         | `group-name`  |
+
+:::note
+Each property will only apply if `TYPE` is set accordingly. For example, `IDENTITY_TENANTS_0_MEMBERS_0_USERNAME` will only function if `TYPE` is set to `USER`.
+:::
+
+When converting the above variables for use in the Helm chart, ensure multiple tenants or members are added as a list. For example, a valid `application.yaml` configuration might look like the following:
+
+```
+identity:
+   tenants:
+    - name: Tenant Name
+      tenantId: tenant-name
+      members:
+        - type: GROUP
+          group-name: "Test Group"
+        - type: USER
+          username: test-user
+        - type: APPLICATION
+          application-id: test-application
+```
+
+## Configure applications
+
+The following variables can be used to configure [applications](/self-managed/identity/user-guide/additional-features/incorporate-applications.md). To add additional applications, increment the `0` value for each variable (for example, `KEYCLOAK_CLIENTS_1_NAME`).
```

Comment:
> ❓ What is the general term we use in the docs client or application? It is anyway scattered (in the configurations) but maybe there is a common term we should use in docs when describing it.

> AFAIK, "application" is a term specific to Identity. I think a client can be an application, but I don't think applications are just clients.

> @romansmirnov Based on your comment, I have adjusted this to application to standardize, but I don't totally understand why all of the variables use "CLIENT" instead of "APPLICATION"?